src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -162,6 +190,22 @@ void GetInteriorVehicleDataRequest::on_event(
 
 GetInteriorVehicleDataRequest::~GetInteriorVehicleDataRequest() {}
 
+std::vector<application_manager::ApplicationSharedPtr>
+GetInteriorVehicleDataRequest::SubscribedToModuleType(
+    const std::string& module_type) {
+  std::vector<application_manager::ApplicationSharedPtr> subscribed_apps;
+  auto apps = RCRPCPlugin::GetRCApplications(application_manager_);
+  for (auto app : apps) {
```

Comment:
@LuxoftAKutsan May be use `for (const auto& app: apps)` ?

Reply:
@AByzhynar QueryInterface is not const method